app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,64 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+data class MovieCardViewState(
+    val imageUrl: String?,
+    val isFavorite: Boolean,
+)
+
+@Composable
+fun MovieCard(
+    movieCardViewState: MovieCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        modifier = Modifier
+            .width(100.dp)
+            .height(150.dp)
+            .padding(5.dp),
+        shape = RoundedCornerShape(10.dp),
+        elevation = 10.dp,
+    ) {
+        Box {
+            AsyncImage(
+                model = movieCardViewState.imageUrl,
+                contentDescription = movieCardViewState.imageUrl,
+                modifier = Modifier.height(150.dp),
```

Comment:
Because size will be set by caller this code won't be correct if the caller decides to set the height of MovieCard to 80.dp or 200.dp. To fix this use Modifier.fillMaxSize()